packages/devtools_app/lib/src/ui/icons.dart

```diff
@@ -243,12 +243,24 @@ class FlutterMaterialIcons {
   }
 }
 
-Image createImageIcon(String url, {double size = defaultIconSize}) {
-  return Image(
-    image: AssetImage(url),
-    height: size,
-    width: size,
-  );
+class AssetImageIcon extends StatelessWidget {
+  const AssetImageIcon(
+      {@required this.url,
```

Comment:
This shouldn't be called URL - it's an asset name - so `assetName`?

Also maybe doesn't need to be a named parameter? It then matches `AssetImage`

Reply:
or probably just `asset` to match the other class